src/components/calltoaction.js

```diff
@@ -29,7 +29,7 @@ const CallToAction = () => (
         </Col>
         <Col>
           <StaticImage src='../images/grid-view-blk.png' alt='grid' />
-          <a rel="noreferrer" target="_blank" href='https://github.com/verizon'>
+          <a target="_blank" href='https://github.com/verizon' rel="noreferrer">
```

Comment:
sorry, but href still isn't first here 🙃

Reply:
djfiooiioawfeio thanks!

Reply:
also you can use `suggestions` to make these easier, (ctrl+g) I'll do one here instead of editing
```suggestion
          <a href="https://github.com/verizon" target="_blank" rel="noreferrer">
```